client/ui/qml/Controls2/TextTypes/ParagraphTextType.qml

```diff
@@ -1,7 +1,7 @@
 import QtQuick
 
 Text {
-    lineHeight: 24
+    lineHeight: LanguageModel.getCurrentLanguageIndex() === 5 ? 30 : 24 // Burmese
```

Comment:
it is better to make a property in languageModel, which will return a value depending on the selected language

Reply:
you are right, that would be better, I'll fix it.